playbooks/roles/l2tp-ipsec/tasks/main.yml

```diff
@@ -14,16 +14,15 @@
     install_recommends: no
 
 - name: Retrieve the Libreswan source code
-  get_url:
-    # This CDN mirrors the Libreswan source code, and helps
-    # mitigate connection errors that were occurring when
-    # using the project's official download location.
-    url: https://d25kfp60e9u1dw.cloudfront.net/{{ libreswan_source_filename }}
```

Comment:
Can you preserve this comment about the CDN mirrors? It seems like it could be moved to `playbooks/roles/l2tp-ipsec/vars/main.yml` above the `libreswan_download_urls`.